packages/core/src/virtual-entrypoint.ts

```diff
@@ -154,7 +154,6 @@ export function renderEntrypoint(
 const entryTemplate = compile(`
 import { importSync as i, macroCondition, getGlobalConfig } from '@embroider/macros';
 let w = window;
```

Comment:
I wonder what the benefit of having window aliased like this is now that we've removed the `let d = w.define;` 🤔